R/interface.R

```diff
@@ -289,9 +289,16 @@ sourceExamples <- function(file){
 rcppoctave <- function(..., plugin = 'RcppOctave'){
 	if( !requireNamespace('inline') )
 		stop("Package 'inline' is required to define RcppOctave functions.")
-	#qlibrary(inline)
-	#rcpp(..., plugin = plugin)
-	cxxfunction(..., plugin = plugin)
+	#qlibrary('inline')
```

Comment:
I had to google `qlibrary` and must have gotten the wrong idea about it. I see now that `pkgmaker` installs easily and is also mentioned in the package `DESCRIPTION`.

I'll undo this edit.